test/continuum_absorption.jl

```diff
@@ -130,89 +131,24 @@ function _Hminus_bf_cross_section_Gray(λ::Real)
     λ5 = λ*λ4
     λ6 = λ*λ5
 
-    αbf_H⁻ = (1.99654 - 1.18267e-5 * λ + 2.64243e-6 * λ2 - 4.40524e-10 * λ3 + 3.23992e-14 * λ4
-              - 1.39568e-18 * λ5 + 2.78701e-23 * λ6)
-    αbf_H⁻
+    1e-18 * (1.99654 - 1.18267e-5 * λ + 2.64243e-6 * λ2 - 4.40524e-10 * λ3 + 3.23992e-14 * λ4
+     - 1.39568e-18 * λ5 + 2.78701e-23 * λ6)
 end
 
-function check_Hminus_bf_values(target_precision = 0.002, verbose = true)
-    # compare the tabulated data from Wishart (1979) against the polynomial fits from Gray (2005)
-    
-    _bf_table = Korg.ContinuumAbsorption._Hminus_bf_table
-    _bf_λ_vals = view(_bf_table, :, 1)
-    _bf_α_vals = view(_bf_table, :, 2)
-
-    comp_λ_vals = view(_bf_λ_vals, 5:56)
-    comp_α_vals = view(_bf_α_vals, 5:56)
-    @assert comp_λ_vals[1] == 2250.0
-    @assert comp_λ_vals[end] == 15000.0
-
-    coefs = _Hminus_bf_cross_section_Gray.(comp_λ_vals)
-    precision = abs.(coefs .- comp_α_vals)./comp_α_vals
-    max_err, max_err_ind = findmax(precision)
-
-    if max_err < target_precision
-        return true
-    else
-        if verbose
-            println("There is a problem. The max error of polynomial should be: ",
-                    target_precision*100, "%.")
-            println("Max error is: ", 100 * max_err,
-                    "% at λ = ", comp_λ_vals[max_err_ind], " Å.")
-        end
-        return false
+@testset "H⁻ bound-free absorption" begin
+    @testset "comparison to Gray" begin
+        # compare the tabulated data from McLaughlin (2017) to the polynomial fits from Gray (2005)
+        λs = 2250 : 1.0 : 15000
+        νs = Korg.c_cgs ./ (λs * 1e-8)
+        grayvals = _Hminus_bf_cross_section_Gray.(λs)
+        korgvals = Korg.ContinuumAbsorption._Hminus_bf_cross_section.(νs)
+        @test assert_allclose_grid(korgvals, grayvals, [("λ", λs, "Å")], rtol=0.02)
     end
-end
-
 
-@testset "H⁻ bound-free absorption" begin
-    @test check_Hminus_bf_values(0.0025)
     @testset "Gray (2005) Fig 8.5$panel comparison" for panel in ["a", "b", "c"]
         calculated, ref = Gray_opac_compare.Gray05_comparison_vals(panel,"Hminus_bf")
         @test all(calculated .≥ 0.0)
-        @test all(abs.(calculated - ref) .≤ Gray_opac_compare.Gray05_atols[panel])
-    end
```

Comment:
It might be nice to add some version of this test back in, just to check that we get a sensible non-zero value between the lowest tabulated frequency from McLaughlin+ 2017 and the ionization energy. But, it's definitely not necessary if you don't want to do it (at the very least, we should probably check that it works manually)

Reply:
To clarify, I was talking about the extreme wavelength test.

Reply:
This is a good idea.  I've written a new test.